### PR TITLE
Revamp swagger ui dependencies

### PR DESCRIPTION
**Pull request by yaooqinn**
### _Why are the changes needed?_

This PR mainly removes the swagger-scala-module which is currently fine to remove. It also introduces transitive deps with
`-jakarta` which could introduce jar conflicts like ,

```scala
Caused by: java.lang.NoClassDefFoundError: jakarta/xml/bind/annotation/XmlElement
	at io.swagger.v3.core.jackson.SwaggerAnnotationIntrospector.hasRequiredMarker(SwaggerAnnotationIntrospector.java:29)
	at com.fasterxml.jackson.databind.introspect.AnnotationIntrospectorPair.hasRequiredMarker(AnnotationIntrospectorPair.java:319)
	at com.fasterxml.jackson.databind.introspect.AnnotationIntrospectorPair.hasRequiredMarker(AnnotationIntrospectorPair.java:319)
	at com.fasterxml.jackson.databind.introspect.AnnotationIntrospectorPair.hasRequiredMarker(AnnotationIntrospectorPair.java:319)
	at com.fasterxml.jackson.databind.introspect.POJOPropertyBuilder.getMetadata(POJOPropertyBuilder.java:229)
```
### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by yaooqinn** [comment]
cc @pan3793 @cfmcgrady

File: pom.xml [hunk 1]
```diff
@@ -670,17 +669,10 @@
                 <groupId>io.swagger.core.v3</groupId>
                 <artifactId>swagger-jaxrs2</artifactId>
                 <version>${swagger.version}</version>
-            </dependency>
-
-            <!-- ref: https://github.com/swagger-akka-http/swagger-scala-module -->
-            <dependency>
-                <groupId>com.github.swagger-akka-http</groupId>
-                <artifactId>swagger-scala-module_${scala.binary.version}</artifactId>
-                <version>${swagger.scala.module.version}</version>
```
Comment (ulysses-you) [inline pom.xml:679]: Is it we do not use `Option` type as the value of annotation, so we can remove this dependency safely ?

Comment (yaooqinn) [reply pom.xml:679]: not quite sure

**Comment by pan3793** [comment]
Always happy to see reducing dependencies

**Review by ulysses-you — APPROVED** [review]

**Comment by ulysses-you** [comment]
thanks, merging to master

**Comment by cfmcgrady** [comment]
Sorry for the later review, @yaooqinn 

> This PR mainly removes the swagger-scala-module which is currently fine to remove. 

This is a breaking change for openapi. `swagger-scala-module` is a library that automatically generates a schema from a Scala case class.

After this PR

```json
  "components" : {
    "schemas" : {
      "OpActionRequest" : {
        "type" : "object"
      },
      "KyuubiOperationEvent" : {
        "type" : "object"
      },
      "OperationLog" : {
        "type" : "object"
      },
      "ResultSetMetaData" : {
        "type" : "object"
      },
      "OperationHandle" : {
        "type" : "object",
        "properties" : {
          "hasResultSet" : {
            "type" : "boolean",
            "writeOnly" : true
          }
        }
      },
      "ExecPoolStatistic" : {
        "type" : "object"
      },
      "StatementRequest" : {
        "type" : "object"
      },
      "GetColumnsRequest" : {
        "type" : "object"
      },
      "GetFunctionsRequest" : {
        "type" : "object"
      },
      "InfoDetail" : {
        "type" : "object"
      },
      "GetSchemasRequest" : {
        "type" : "object"
      },
      "GetTablesRequest" : {
        "type" : "object"
      },
      "SessionHandle" : {
        "type" : "object"
      },
      "SessionOpenRequest" : {
        "type" : "object"
      },
      "SessionOpenCount" : {
        "type" : "object"
      },
      "KyuubiSessionEvent" : {
        "type" : "object"
      },
      "SessionList" : {
        "type" : "object"
      }
    }
  }
```

Before this PR

```json
  "components" : {
    "schemas" : {
      "OpActionRequest" : {
        "required" : [ "action" ],
        "type" : "object",
        "properties" : {
          "action" : {
            "type" : "string"
          }
        }
      },
      "KyuubiOperationEvent" : {
        "required" : [ "eventTime", "shouldRunAsync", "statementId" ],
        "type" : "object",
        "properties" : {
          "statementId" : {
            "type" : "string"
          },
          "remoteId" : {
            "type" : "string"
          },
          "statement" : {
            "type" : "string"
          },
          "shouldRunAsync" : {
            "type" : "boolean"
          },
          "state" : {
            "type" : "string"
          },
          "eventTime" : {
            "type" : "integer",
            "format" : "int64"
          },
          "createTime" : {
            "type" : "integer",
            "format" : "int64"
          },
          "startTime" : {
            "type" : "integer",
            "format" : "int64"
          },
          "completeTime" : {
            "type" : "integer",
            "format" : "int64"
          },
          "exception" : {
            "type" : "object",
            "properties" : {
              "cause" : {
                "required" : [ "stackTrace", "suppressed" ],
                "type" : "object",
                "properties" : {
                  "stackTrace" : {
                    "type" : "array",
                    "items" : {
                      "required" : [ "lineNumber" ],
                      "type" : "object",
                      "properties" : {
                        "methodName" : {
                          "type" : "string"
                        },
                        "fileName" : {
                          "type" : "string"
                        },
                        "lineNumber" : {
                          "type" : "integer",
                          "format" : "int32"
                        },
                        "className" : {
                          "type" : "string"
                        },
                        "nativeMethod" : {
                          "type" : "boolean"
                        }
                      }
                    }
                  },
                  "message" : {
                    "type" : "string"
                  },
                  "localizedMessage" : {
                    "type" : "string"
                  },
                  "suppressed" : {
                    "type" : "array",
                    "items" : {
                      "type" : "object",
                      "properties" : {
                        "stackTrace" : {
                          "type" : "array",
                          "items" : {
                            "required" : [ "lineNumber" ],
                            "type" : "object",
                            "properties" : {
                              "methodName" : {
                                "type" : "string"
                              },
                              "fileName" : {
                                "type" : "string"
                              },
                              "lineNumber" : {
                                "type" : "integer",
                                "format" : "int32"
                              },
                              "className" : {
                                "type" : "string"
                              },
                              "nativeMethod" : {
                                "type" : "boolean"
                              }
                            }
                          }
                        },
                        "message" : {
                          "type" : "string"
                        },
                        "localizedMessage" : {
                          "type" : "string"
                        }
                      }
                    }
                  }
                }
              },
              "stackTrace" : {
                "type" : "array",
                "items" : {
                  "required" : [ "lineNumber" ],
                  "type" : "object",
                  "properties" : {
                    "methodName" : {
                      "type" : "string"
                    },
                    "fileName" : {
                      "type" : "string"
                    },
                    "lineNumber" : {
                      "type" : "integer",
                      "format" : "int32"
                    },
                    "className" : {
                      "type" : "string"
                    },
                    "nativeMethod" : {
                      "type" : "boolean"
                    }
                  }
                }
              },
              "message" : {
                "type" : "string"
              },
              "localizedMessage" : {
                "type" : "string"
              },
              "suppressed" : {
                "type" : "array",
                "items" : {
                  "type" : "object",
                  "properties" : {
                    "stackTrace" : {
                      "type" : "array",
                      "items" : {
                        "required" : [ "lineNumber" ],
                        "type" : "object",
                        "properties" : {
                          "methodName" : {
                            "type" : "string"
                          },
                          "fileName" : {
                            "type" : "string"
                          },
                          "lineNumber" : {
                            "type" : "integer",
                            "format" : "int32"
                          },
                          "className" : {
                            "type" : "string"
                          },
                          "nativeMethod" : {
                            "type" : "boolean"
                          }
                        }
                      }
                    },
                    "message" : {
                      "type" : "string"
                    },
                    "localizedMessage" : {
                      "type" : "string"
                    }
                  }
                }
              }
            }
          },
          "sessionId" : {
            "type" : "string"
          },
          "sessionUser" : {
            "type" : "string"
          },
          "eventType" : {
            "type" : "string"
          }
        }
      },
      "OperationLog" : {
        "required" : [ "logRowSet", "rowCount" ],
        "type" : "object",
        "properties" : {
          "logRowSet" : {
            "type" : "array",
            "items" : {
              "type" : "string"
            }
          },
          "rowCount" : {
            "type" : "integer",
            "format" : "int32"
          }
        }
      },
      "SeqString" : {
        "type" : "array",
        "items" : {
          "type" : "string"
        }
      },
      "ColumnDesc" : {
        "required" : [ "columnIndex", "columnName" ],
        "type" : "object",
        "properties" : {
          "columnName" : {
            "type" : "string"
          },
          "dataType" : {
            "type" : "string"
          },
          "columnIndex" : {
            "type" : "integer",
            "format" : "int32"
          },
          "precision" : {
            "type" : "integer",
            "format" : "int32"
          },
          "scale" : {
            "type" : "integer",
            "format" : "int32"
          },
          "comment" : {
            "type" : "string"
          }
        }
      },
      "ResultSetMetaData" : {
        "required" : [ "columns" ],
        "type" : "object",
        "properties" : {
          "columns" : {
            "type" : "array",
            "items" : {
              "$ref" : "#/components/schemas/ColumnDesc"
            }
          }
        }
      },
      "SeqColumnDesc" : {
        "type" : "array",
        "items" : {
          "$ref" : "#/components/schemas/ColumnDesc"
        }
      },
      "HandleIdentifier" : {
        "required" : [ "publicId" ],
        "type" : "object",
        "properties" : {
          "publicId" : {
            "type" : "string",
            "format" : "uuid"
          },
          "secretId" : {
            "type" : "string",
            "format" : "uuid"
          }
        }
      },
      "OperationHandle" : {
        "required" : [ "hasResultSet", "identifier", "protocol", "typ" ],
        "type" : "object",
        "properties" : {
          "identifier" : {
            "$ref" : "#/components/schemas/HandleIdentifier"
          },
          "typ" : {
            "$ref" : "#/components/schemas/Value"
          },
          "protocol" : {
            "type" : "string",
            "enum" : [ "HIVE_CLI_SERVICE_PROTOCOL_V1", "HIVE_CLI_SERVICE_PROTOCOL_V2", "HIVE_CLI_SERVICE_PROTOCOL_V3", "HIVE_CLI_SERVICE_PROTOCOL_V4", "HIVE_CLI_SERVICE_PROTOCOL_V5", "HIVE_CLI_SERVICE_PROTOCOL_V6", "HIVE_CLI_SERVICE_PROTOCOL_V7", "HIVE_CLI_SERVICE_PROTOCOL_V8", "HIVE_CLI_SERVICE_PROTOCOL_V9", "HIVE_CLI_SERVICE_PROTOCOL_V10" ]
          },
          "hasResultSet" : {
            "type" : "boolean",
            "writeOnly" : true
          }
        }
      },
      "Value" : {
        "type" : "object"
      },
      "ExecPoolStatistic" : {
        "required" : [ "execPoolSize" ],
        "type" : "object",
        "properties" : {
          "execPoolSize" : {
            "type" : "integer",
            "format" : "int32"
          },
          "execPoolActiveCount" : {
            "type" : "integer",
            "format" : "int32"
          }
        }
      },
      "StatementRequest" : {
        "required" : [ "queryTimeout", "runAsync", "statement" ],
        "type" : "object",
        "properties" : {
          "statement" : {
            "type" : "string"
          },
          "runAsync" : {
            "type" : "boolean"
          },
          "queryTimeout" : {
            "type" : "integer",
            "format" : "int64"
          }
        }
      },
      "GetColumnsRequest" : {
        "required" : [ "catalogName" ],
        "type" : "object",
        "properties" : {
          "catalogName" : {
            "type" : "string"
          },
          "schemaName" : {
            "type" : "string"
          },
          "tableName" : {
            "type" : "string"
          },
          "columnName" : {
            "type" : "string"
          }
        }
      },
      "GetFunctionsRequest" : {
        "required" : [ "catalogName" ],
        "type" : "object",
        "properties" : {
          "catalogName" : {
            "type" : "string"
          },
          "schemaName" : {
            "type" : "string"
          },
          "functionName" : {
            "type" : "string"
          }
        }
      },
      "InfoDetail" : {
        "required" : [ "infoType" ],
        "type" : "object",
        "properties" : {
          "infoType" : {
            "type" : "string"
          },
          "infoValue" : {
            "type" : "string"
          }
        }
      },
      "GetSchemasRequest" : {
        "required" : [ "catalogName" ],
        "type" : "object",
        "properties" : {
          "catalogName" : {
            "type" : "string"
          },
          "schemaName" : {
            "type" : "string"
          }
        }
      },
      "GetTablesRequest" : {
        "required" : [ "catalogName", "tableTypes" ],
        "type" : "object",
        "properties" : {
          "catalogName" : {
            "type" : "string"
          },
          "schemaName" : {
            "type" : "string"
          },
          "tableName" : {
            "type" : "string"
          },
          "tableTypes" : {
            "type" : "array",
            "items" : {
              "type" : "string"
            }
          }
        }
      },
      "SessionHandle" : {
        "required" : [ "identifier", "protocol" ],
        "type" : "object",
        "properties" : {
          "identifier" : {
            "$ref" : "#/components/schemas/HandleIdentifier"
          },
          "protocol" : {
            "type" : "string",
            "enum" : [ "HIVE_CLI_SERVICE_PROTOCOL_V1", "HIVE_CLI_SERVICE_PROTOCOL_V2", "HIVE_CLI_SERVICE_PROTOCOL_V3", "HIVE_CLI_SERVICE_PROTOCOL_V4", "HIVE_CLI_SERVICE_PROTOCOL_V5", "HIVE_CLI_SERVICE_PROTOCOL_V6", "HIVE_CLI_SERVICE_PROTOCOL_V7", "HIVE_CLI_SERVICE_PROTOCOL_V8", "HIVE_CLI_SERVICE_PROTOCOL_V9", "HIVE_CLI_SERVICE_PROTOCOL_V10" ]
          }
        }
      },
      "MapStringString" : {
        "type" : "object",
        "additionalProperties" : {
          "type" : "string"
        }
      },
      "SessionOpenRequest" : {
        "required" : [ "configs", "protocolVersion", "user" ],
        "type" : "object",
        "properties" : {
          "protocolVersion" : {
            "type" : "integer",
            "format" : "int32"
          },
          "user" : {
            "type" : "string"
          },
          "password" : {
            "type" : "string"
          },
          "ipAddr" : {
            "type" : "string"
          },
          "configs" : {
            "type" : "object",
            "additionalProperties" : {
              "type" : "string"
            }
          }
        }
      },
      "SessionOpenCount" : {
        "required" : [ "openSessionCount" ],
        "type" : "object",
        "properties" : {
          "openSessionCount" : {
            "type" : "integer",
            "format" : "int32"
          }
        }
      },
      "KyuubiSessionEvent" : {
        "required" : [ "clientVersion", "conf", "sessionId", "startTime" ],
        "type" : "object",
        "properties" : {
          "sessionId" : {
            "type" : "string"
          },
          "clientVersion" : {
            "type" : "integer",
            "format" : "int32"
          },
          "sessionName" : {
            "type" : "string"
          },
          "user" : {
            "type" : "string"
          },
          "clientIP" : {
            "type" : "string"
          },
          "serverIP" : {
            "type" : "string"
          },
          "conf" : {
            "type" : "object",
            "additionalProperties" : {
              "type" : "string"
            }
          },
          "startTime" : {
            "type" : "integer",
            "format" : "int64"
          },
          "remoteSessionId" : {
            "type" : "string"
          },
          "engineId" : {
            "type" : "string"
          },
          "openedTime" : {
            "type" : "integer",
            "format" : "int64"
          },
          "endTime" : {
            "type" : "integer",
            "format" : "int64"
          },
          "totalOperations" : {
            "type" : "integer",
            "format" : "int32"
          },
          "eventType" : {
            "type" : "string"
          }
        }
      },
      "SeqSessionOverview" : {
        "type" : "array",
        "items" : {
          "$ref" : "#/components/schemas/SessionOverview"
        }
      },
      "SessionList" : {
        "required" : [ "sessionList" ],
        "type" : "object",
        "properties" : {
          "sessionList" : {
            "type" : "array",
            "items" : {
              "$ref" : "#/components/schemas/SessionOverview"
            }
          }
        }
      },
      "SessionOverview" : {
        "required" : [ "createTime", "user" ],
        "type" : "object",
        "properties" : {
          "user" : {
            "type" : "string"
          },
          "ipAddr" : {
            "type" : "string"
          },
          "createTime" : {
            "type" : "integer",
            "format" : "int64"
          },
          "sessionHandle" : {
            "$ref" : "#/components/schemas/SessionHandle"
          }
        }
      }
    }
  }
```

**Comment by yaooqinn** [comment]
> ```json
> components
> ```



> Sorry for the later review, @yaooqinn
> 
> > This PR mainly removes the swagger-scala-module which is currently fine to remove.
> 
> This is a breaking change for openapi. `swagger-scala-module` is a library that automatically generates a schema from a Scala case class.
> 
> After this PR
> 
> ```json
>   "components" : {
>     "schemas" : {
>       "OpActionRequest" : {
>         "type" : "object"
>       },
>       "KyuubiOperationEvent" : {
>         "type" : "object"
>       },
>       "OperationLog" : {
>         "type" : "object"
>       },
>       "ResultSetMetaData" : {
>         "type" : "object"
>       },
>       "OperationHandle" : {
>         "type" : "object",
>         "properties" : {
>           "hasResultSet" : {
>             "type" : "boolean",
>             "writeOnly" : true
>           }
>         }
>       },
>       "ExecPoolStatistic" : {
>         "type" : "object"
>       },
>       "StatementRequest" : {
>         "type" : "object"
>       },
>       "GetColumnsRequest" : {
>         "type" : "object"
>       },
>       "GetFunctionsRequest" : {
>         "type" : "object"
>       },
>       "InfoDetail" : {
>         "type" : "object"
>       },
>       "GetSchemasRequest" : {
>         "type" : "object"
>       },
>       "GetTablesRequest" : {
>         "type" : "object"
>       },
>       "SessionHandle" : {
>         "type" : "object"
>       },
>       "SessionOpenRequest" : {
>         "type" : "object"
>       },
>       "SessionOpenCount" : {
>         "type" : "object"
>       },
>       "KyuubiSessionEvent" : {
>         "type" : "object"
>       },
>       "SessionList" : {
>         "type" : "object"
>       }
>     }
>   }
> ```
> 
> Before this PR
> 
> ```json
>   "components" : {
>     "schemas" : {
>       "OpActionRequest" : {
>         "required" : [ "action" ],
>         "type" : "object",
>         "properties" : {
>           "action" : {
>             "type" : "string"
>           }
>         }
>       },
>       "KyuubiOperationEvent" : {
>         "required" : [ "eventTime", "shouldRunAsync", "statementId" ],
>         "type" : "object",
>         "properties" : {
>           "statementId" : {
>             "type" : "string"
>           },
>           "remoteId" : {
>             "type" : "string"
>           },
>           "statement" : {
>             "type" : "string"
>           },
>           "shouldRunAsync" : {
>             "type" : "boolean"
>           },
>           "state" : {
>             "type" : "string"
>           },
>           "eventTime" : {
>             "type" : "integer",
>             "format" : "int64"
>           },
>           "createTime" : {
>             "type" : "integer",
>             "format" : "int64"
>           },
>           "startTime" : {
>             "type" : "integer",
>             "format" : "int64"
>           },
>           "completeTime" : {
>             "type" : "integer",
>             "format" : "int64"
>           },
>           "exception" : {
>             "type" : "object",
>             "properties" : {
>               "cause" : {
>                 "required" : [ "stackTrace", "suppressed" ],
>                 "type" : "object",
>                 "properties" : {
>                   "stackTrace" : {
>                     "type" : "array",
>                     "items" : {
>                       "required" : [ "lineNumber" ],
>                       "type" : "object",
>                       "properties" : {
>                         "methodName" : {
>                           "type" : "string"
>                         },
>                         "fileName" : {
>                           "type" : "string"
>                         },
>                         "lineNumber" : {
>                           "type" : "integer",
>                           "format" : "int32"
>                         },
>                         "className" : {
>                           "type" : "string"
>                         },
>                         "nativeMethod" : {
>                           "type" : "boolean"
>                         }
>                       }
>                     }
>                   },
>                   "message" : {
>                     "type" : "string"
>                   },
>                   "localizedMessage" : {
>                     "type" : "string"
>                   },
>                   "suppressed" : {
>                     "type" : "array",
>                     "items" : {
>                       "type" : "object",
>                       "properties" : {
>                         "stackTrace" : {
>                           "type" : "array",
>                           "items" : {
>                             "required" : [ "lineNumber" ],
>                             "type" : "object",
>                             "properties" : {
>                               "methodName" : {
>                                 "type" : "string"
>                               },
>                               "fileName" : {
>                                 "type" : "string"
>                               },
>                               "lineNumber" : {
>                                 "type" : "integer",
>                                 "format" : "int32"
>                               },
>                               "className" : {
>                                 "type" : "string"
>                               },
>                               "nativeMethod" : {
>                                 "type" : "boolean"
>                               }
>                             }
>                           }
>                         },
>                         "message" : {
>                           "type" : "string"
>                         },
>                         "localizedMessage" : {
>                           "type" : "string"
>                         }
>                       }
>                     }
>                   }
>                 }
>               },
>               "stackTrace" : {
>                 "type" : "array",
>                 "items" : {
>                   "required" : [ "lineNumber" ],
>                   "type" : "object",
>                   "properties" : {
>                     "methodName" : {
>                       "type" : "string"
>                     },
>                     "fileName" : {
>                       "type" : "string"
>                     },
>                     "lineNumber" : {
>                       "type" : "integer",
>                       "format" : "int32"
>                     },
>                     "className" : {
>                       "type" : "string"
>                     },
>                     "nativeMethod" : {
>                       "type" : "boolean"
>                     }
>                   }
>                 }
>               },
>               "message" : {
>                 "type" : "string"
>               },
>               "localizedMessage" : {
>                 "type" : "string"
>               },
>               "suppressed" : {
>                 "type" : "array",
>                 "items" : {
>                   "type" : "object",
>                   "properties" : {
>                     "stackTrace" : {
>                       "type" : "array",
>                       "items" : {
>                         "required" : [ "lineNumber" ],
>                         "type" : "object",
>                         "properties" : {
>                           "methodName" : {
>                             "type" : "string"
>                           },
>                           "fileName" : {
>                             "type" : "string"
>                           },
>                           "lineNumber" : {
>                             "type" : "integer",
>                             "format" : "int32"
>                           },
>                           "className" : {
>                             "type" : "string"
>                           },
>                           "nativeMethod" : {
>                             "type" : "boolean"
>                           }
>                         }
>                       }
>                     },
>                     "message" : {
>                       "type" : "string"
>                     },
>                     "localizedMessage" : {
>                       "type" : "string"
>                     }
>                   }
>                 }
>               }
>             }
>           },
>           "sessionId" : {
>             "type" : "string"
>           },
>           "sessionUser" : {
>             "type" : "string"
>           },
>           "eventType" : {
>             "type" : "string"
>           }
>         }
>       },
>       "OperationLog" : {
>         "required" : [ "logRowSet", "rowCount" ],
>         "type" : "object",
>         "properties" : {
>           "logRowSet" : {
>             "type" : "array",
>             "items" : {
>               "type" : "string"
>             }
>           },
>           "rowCount" : {
>             "type" : "integer",
>             "format" : "int32"
>           }
>         }
>       },
>       "SeqString" : {
>         "type" : "array",
>         "items" : {
>           "type" : "string"
>         }
>       },
>       "ColumnDesc" : {
>         "required" : [ "columnIndex", "columnName" ],
>         "type" : "object",
>         "properties" : {
>           "columnName" : {
>             "type" : "string"
>           },
>           "dataType" : {
>             "type" : "string"
>           },
>           "columnIndex" : {
>             "type" : "integer",
>             "format" : "int32"
>           },
>           "precision" : {
>             "type" : "integer",
>             "format" : "int32"
>           },
>           "scale" : {
>             "type" : "integer",
>             "format" : "int32"
>           },
>           "comment" : {
>             "type" : "string"
>           }
>         }
>       },
>       "ResultSetMetaData" : {
>         "required" : [ "columns" ],
>         "type" : "object",
>         "properties" : {
>           "columns" : {
>             "type" : "array",
>             "items" : {
>               "$ref" : "#/components/schemas/ColumnDesc"
>             }
>           }
>         }
>       },
>       "SeqColumnDesc" : {
>         "type" : "array",
>         "items" : {
>           "$ref" : "#/components/schemas/ColumnDesc"
>         }
>       },
>       "HandleIdentifier" : {
>         "required" : [ "publicId" ],
>         "type" : "object",
>         "properties" : {
>           "publicId" : {
>             "type" : "string",
>             "format" : "uuid"
>           },
>           "secretId" : {
>             "type" : "string",
>             "format" : "uuid"
>           }
>         }
>       },
>       "OperationHandle" : {
>         "required" : [ "hasResultSet", "identifier", "protocol", "typ" ],
>         "type" : "object",
>         "properties" : {
>           "identifier" : {
>             "$ref" : "#/components/schemas/HandleIdentifier"
>           },
>           "typ" : {
>             "$ref" : "#/components/schemas/Value"
>           },
>           "protocol" : {
>             "type" : "string",
>             "enum" : [ "HIVE_CLI_SERVICE_PROTOCOL_V1", "HIVE_CLI_SERVICE_PROTOCOL_V2", "HIVE_CLI_SERVICE_PROTOCOL_V3", "HIVE_CLI_SERVICE_PROTOCOL_V4", "HIVE_CLI_SERVICE_PROTOCOL_V5", "HIVE_CLI_SERVICE_PROTOCOL_V6", "HIVE_CLI_SERVICE_PROTOCOL_V7", "HIVE_CLI_SERVICE_PROTOCOL_V8", "HIVE_CLI_SERVICE_PROTOCOL_V9", "HIVE_CLI_SERVICE_PROTOCOL_V10" ]
>           },
>           "hasResultSet" : {
>             "type" : "boolean",
>             "writeOnly" : true
>           }
>         }
>       },
>       "Value" : {
>         "type" : "object"
>       },
>       "ExecPoolStatistic" : {
>         "required" : [ "execPoolSize" ],
>         "type" : "object",
>         "properties" : {
>           "execPoolSize" : {
>             "type" : "integer",
>             "format" : "int32"
>           },
>           "execPoolActiveCount" : {
>             "type" : "integer",
>             "format" : "int32"
>           }
>         }
>       },
>       "StatementRequest" : {
>         "required" : [ "queryTimeout", "runAsync", "statement" ],
>         "type" : "object",
>         "properties" : {
>           "statement" : {
>             "type" : "string"
>           },
>           "runAsync" : {
>             "type" : "boolean"
>           },
>           "queryTimeout" : {
>             "type" : "integer",
>             "format" : "int64"
>           }
>         }
>       },
>       "GetColumnsRequest" : {
>         "required" : [ "catalogName" ],
>         "type" : "object",
>         "properties" : {
>           "catalogName" : {
>             "type" : "string"
>           },
>           "schemaName" : {
>             "type" : "string"
>           },
>           "tableName" : {
>             "type" : "string"
>           },
>           "columnName" : {
>             "type" : "string"
>           }
>         }
>       },
>       "GetFunctionsRequest" : {
>         "required" : [ "catalogName" ],
>         "type" : "object",
>         "properties" : {
>           "catalogName" : {
>             "type" : "string"
>           },
>           "schemaName" : {
>             "type" : "string"
>           },
>           "functionName" : {
>             "type" : "string"
>           }
>         }
>       },
>       "InfoDetail" : {
>         "required" : [ "infoType" ],
>         "type" : "object",
>         "properties" : {
>           "infoType" : {
>             "type" : "string"
>           },
>           "infoValue" : {
>             "type" : "string"
>           }
>         }
>       },
>       "GetSchemasRequest" : {
>         "required" : [ "catalogName" ],
>         "type" : "object",
>         "properties" : {
>           "catalogName" : {
>             "type" : "string"
>           },
>           "schemaName" : {
>             "type" : "string"
>           }
>         }
>       },
>       "GetTablesRequest" : {
>         "required" : [ "catalogName", "tableTypes" ],
>         "type" : "object",
>         "properties" : {
>           "catalogName" : {
>             "type" : "string"
>           },
>           "schemaName" : {
>             "type" : "string"
>           },
>           "tableName" : {
>             "type" : "string"
>           },
>           "tableTypes" : {
>             "type" : "array",
>             "items" : {
>               "type" : "string"
>             }
>           }
>         }
>       },
>       "SessionHandle" : {
>         "required" : [ "identifier", "protocol" ],
>         "type" : "object",
>         "properties" : {
>           "identifier" : {
>             "$ref" : "#/components/schemas/HandleIdentifier"
>           },
>           "protocol" : {
>             "type" : "string",
>             "enum" : [ "HIVE_CLI_SERVICE_PROTOCOL_V1", "HIVE_CLI_SERVICE_PROTOCOL_V2", "HIVE_CLI_SERVICE_PROTOCOL_V3", "HIVE_CLI_SERVICE_PROTOCOL_V4", "HIVE_CLI_SERVICE_PROTOCOL_V5", "HIVE_CLI_SERVICE_PROTOCOL_V6", "HIVE_CLI_SERVICE_PROTOCOL_V7", "HIVE_CLI_SERVICE_PROTOCOL_V8", "HIVE_CLI_SERVICE_PROTOCOL_V9", "HIVE_CLI_SERVICE_PROTOCOL_V10" ]
>           }
>         }
>       },
>       "MapStringString" : {
>         "type" : "object",
>         "additionalProperties" : {
>           "type" : "string"
>         }
>       },
>       "SessionOpenRequest" : {
>         "required" : [ "configs", "protocolVersion", "user" ],
>         "type" : "object",
>         "properties" : {
>           "protocolVersion" : {
>             "type" : "integer",
>             "format" : "int32"
>           },
>           "user" : {
>             "type" : "string"
>           },
>           "password" : {
>             "type" : "string"
>           },
>           "ipAddr" : {
>             "type" : "string"
>           },
>           "configs" : {
>             "type" : "object",
>             "additionalProperties" : {
>               "type" : "string"
>             }
>           }
>         }
>       },
>       "SessionOpenCount" : {
>         "required" : [ "openSessionCount" ],
>         "type" : "object",
>         "properties" : {
>           "openSessionCount" : {
>             "type" : "integer",
>             "format" : "int32"
>           }
>         }
>       },
>       "KyuubiSessionEvent" : {
>         "required" : [ "clientVersion", "conf", "sessionId", "startTime" ],
>         "type" : "object",
>         "properties" : {
>           "sessionId" : {
>             "type" : "string"
>           },
>           "clientVersion" : {
>             "type" : "integer",
>             "format" : "int32"
>           },
>           "sessionName" : {
>             "type" : "string"
>           },
>           "user" : {
>             "type" : "string"
>           },
>           "clientIP" : {
>             "type" : "string"
>           },
>           "serverIP" : {
>             "type" : "string"
>           },
>           "conf" : {
>             "type" : "object",
>             "additionalProperties" : {
>               "type" : "string"
>             }
>           },
>           "startTime" : {
>             "type" : "integer",
>             "format" : "int64"
>           },
>           "remoteSessionId" : {
>             "type" : "string"
>           },
>           "engineId" : {
>             "type" : "string"
>           },
>           "openedTime" : {
>             "type" : "integer",
>             "format" : "int64"
>           },
>           "endTime" : {
>             "type" : "integer",
>             "format" : "int64"
>           },
>           "totalOperations" : {
>             "type" : "integer",
>             "format" : "int32"
>           },
>           "eventType" : {
>             "type" : "string"
>           }
>         }
>       },
>       "SeqSessionOverview" : {
>         "type" : "array",
>         "items" : {
>           "$ref" : "#/components/schemas/SessionOverview"
>         }
>       },
>       "SessionList" : {
>         "required" : [ "sessionList" ],
>         "type" : "object",
>         "properties" : {
>           "sessionList" : {
>             "type" : "array",
>             "items" : {
>               "$ref" : "#/components/schemas/SessionOverview"
>             }
>           }
>         }
>       },
>       "SessionOverview" : {
>         "required" : [ "createTime", "user" ],
>         "type" : "object",
>         "properties" : {
>           "user" : {
>             "type" : "string"
>           },
>           "ipAddr" : {
>             "type" : "string"
>           },
>           "createTime" : {
>             "type" : "integer",
>             "format" : "int64"
>           },
>           "sessionHandle" : {
>             "$ref" : "#/components/schemas/SessionHandle"
>           }
>         }
>       }
>     }
>   }
> ```

can we inline some logic for that, that module seems on have one class

**Pull request closed without merging**